### PR TITLE
torch frontend comparison ops

### PR DESCRIPTION
**Pull request by AnnaTz**

### CODE REVIEW

**Comment by AnnaTz** [comment]
Closes #5030

File: ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py [hunk 1]
```diff
@@ -726,3 +726,85 @@ def test_torch_msort(
...
+    k,
+    keepdim,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, input, dim = dtype_input_axis
+    input = np.asarray(input, dtype=input_dtype)
```
Comment (simonetgordon) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py:793]: Error with input data dtype seemingly, could from an assignment issue in line 792.
```
>       input = np.asarray(input, dtype=input_dtype)
E       TypeError: Field elements must be 2- or 3-tuples, got ''float64''
```

File: ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py [hunk 2]
```diff
@@ -726,3 +726,85 @@ def test_torch_msort(
...
+    dtype_and_inputs=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.maximum"
+    ),
+)
+def test_torch_maximum(
```
Comment (simonetgordon) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py:743]: Seems to be passing just fine, nice!

**Review by simonetgordon — CHANGES_REQUESTED** [review]

**Comment by AnnaTz** [comment]
Close #5029

**Review by simonetgordon — APPROVED** [review]
lgtm!

**Pull request merged**